### DISCUSSION TITLE
login user password

### DISCUSSION THREAD

**Discussion by wanganzhi1986** [astutic/Acharya#7] [Q&A] [answered]
how to login？ what is password

**Comment by vimal-m** [comment, marked as answer]
For acharya versions < 0.3.0-alpha the password is autogenerated when acharya is run for the first time. If you have run acharya and didn't copy the password then you can reset the password by running the following command

<acharya_cmd> tasks users -r for eg. in Linux the following command can be run
`
./acharya.0.2.0-alpha.linux.amd64 tasks users -r
`

And it will print on screen a system-generated password like this.

> User Successfully created. Please use admin@localhost and pouring.fox to login.

In this eg. `pouring.fox` would be the password.

This design has been modified in acharya version >0.3.0-alpha and now the UI itself asks the user to set a password on first run. Unfortunately, our doc site got updated before the release of 0.3.0-alpha and that might have led you to this problem. Sorry for the trouble.
